src/mscorlib/ref/mscorlib.cs

```diff
@@ -5730,7 +5730,7 @@ public partial class CultureInfo : System.ICloneable, System.IFormatProvider
         public static System.Globalization.CultureInfo GetCultureInfo(string name) { throw null; }
         public static System.Globalization.CultureInfo GetCultureInfo(string name, string altName) { throw null; }
         public static System.Globalization.CultureInfo GetCultureInfoByIetfLanguageTag(string name) { throw null; }
-        public static System.Globalization.CultureInfo[] GetCultures(System.Globalization.CultureTypes types) { throw null; }
```

Comment:
could you add a small comment telling why we are commenting this API and the enum? and we need to open issue tack understanding what is going on.